tests/test_word_bank.py

```diff
@@ -7,10 +7,14 @@
 
```

Comment:
I'd add some tests for different user levels + edge cases.

I'd suggest using `pytest.mark.parametrize`, to save some lines. You can check an example out in _dailyWord-bot/tests/test_utils.py_